.github/workflows/qa-tip-tracking.yml

```diff
@@ -45,6 +45,40 @@ jobs:
       run: |
         python3 $ERIGON_QA_PATH/test_system/db-producer/pause_production.py || true
 
+    - name: Run previous Erigon version and wait for sync (stabilization step)
+      id: pre_test_step
+      run: |
+        set +e # Disable exit on error
+        
+        # Launch the testbed Erigon instance & test its ability to maintain sync for 2 minutes
+        python3 $ERIGON_QA_PATH/test_system/qa-tests/tip-tracking/run_and_check_tip_tracking.py \
+          ${{ env.ERIGON_TESTBED_DATA_DIR }}/../ $ERIGON_TESTBED_DATA_DIR 120 $TOTAL_TIME_SECONDS Erigon3 $CHAIN
```

Comment:
If we want to update the pre-built database, we need to use ERIGON_REFERENCE_DATA_DIR and not ERIGON_TESTBED_DATA_DIR, which is empty at this step.
Otherwise, if we only want to update the pre-built that will be used for the test, leaving the original one untouched, we need to postpone this step after the save_chaindata_step

Reply:
We do need to update the pre-built database, so IMO the `pre_test_step` must stay here. You're right about the `ERIGON_REFERENCE_DATA_DIR `, good catch!